docker/Dockerfile

```diff
@@ -1,24 +1,34 @@
-FROM centos:7 as stage1
+FROM ubuntu:22.04 as stage1
 
 COPY setup-hw /usr/local/bin/setup-hw
 COPY monitor-hw /usr/local/sbin/monitor-hw
 COPY collector /usr/local/sbin/collector
 COPY setup-apply-firmware /usr/local/sbin/setup-apply-firmware
 COPY setup-isoreboot /usr/local/sbin/setup-isoreboot
-
+COPY systemctl-dummy /usr/bin/systemctl
 CMD ["/usr/local/sbin/monitor-hw"]
 
-
 # Install vendor tools
 FROM stage1
 
-ARG DSU_VERSION=21.11.26
+# See https://linux.dell.com/repo/community/openmanage/
+ARG OMSA_VERSION=11000
+ARG UBUNTU_VERSION=jammy
 
-RUN curl -sf -o bootstrap.cgi https://linux.dell.com/repo/hardware/DSU_${DSU_VERSION}/bootstrap.cgi \
-    && bash bootstrap.cgi \
-    && rm bootstrap.cgi \
-    && yum update -y \
-    && yum install -y syscfg srvadmin-idracadm7 dmidecode \
-    && yum clean all
+# Setup DELL repository
+RUN apt update -y && apt-get install wget pgp apt-utils -y \
+    && echo "deb http://linux.dell.com/repo/community/openmanage/${OMSA_VERSION}/${UBUNTU_VERSION} ${UBUNTU_VERSION} main" | tee -a /etc/apt/sources.list.d/linux.dell.com.sources.list \
+    && wget https://linux.dell.com/repo/pgp_pubkeys/0x1285491434D8786F.asc \
+    && apt-key add 0x1285491434D8786F.asc \
+    && rm 0x1285491434D8786F.asc
 
+# Install DELL's tools
+# the installation of dcism will failed due to no GUI in the container. Therefore added "exit 0" to avoid stopping job.
+RUN apt-get update -y \
+    && chmod +x /usr/bin/systemctl \
```

Comment:
It would be good to set execute bits of `systemctl-dummy` in the repository from the first.

Reply:
I follow your comment.